static/usage/select/customization/interface-options/demo.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Select - Interface Options</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+
+  <style>
+    .container {
+      align-items: flex-start;
+    }
+
+    .container > ion-list {
+      min-width: 400px;
+    }
+  </style>
+</head>
+
+<body>
+  <ion-app>
```

Comment:
Rather than align all of the items to the top, are we able to make the demo height large enough such that the popover can never go offscreen? I think the issue is that it's trying to find the best positioning (above or below), but no matter what it's going to get cut off when the items are centered.

![image](https://user-images.githubusercontent.com/2721089/166538849-d0258f14-d5ee-4552-9098-18843644d24c.png)